### DISCUSSION TITLE
Welcome to the AngularFire discussion board!

### DISCUSSION THREAD

**Discussion by jamesdaniels** [angular/angularfire#2707] [General]
We have Github Discussions now! Figured this would be a great spot for the community to collaborate; show off cool things that they've build; and discuss Angular, Firebase, and naturally AngularFire!

`console.log('Hello World!');`

* Use our [Q&A board](https://github.com/angular/angularfire/discussions?discussions_q=category%3AQ%26A) to ask technical questions and get help troubleshooting your app. I'd also suggest asking on [StackOverflow](https://stackoverflow.com/questions/tagged/angularfire2). If you do cross-post please include a link to your question on other boards, so it's easier to cross-post answers. __As always a minimal reproduction on StackBlitz is very much appreciated and recommended!__
* The [Ideas board](https://github.com/angular/angularfire/discussions?discussions_q=category%3AIdeas) will be the primary place we discuss feature requests, proposals, and roadmaps. We'll only allow an issue for a feature request once the feature / proposal is accepted, well defined, a small enough chuck to be worth tracking that way, or could use assistance from the community.
* General and show and tell are pretty self explanatory

This is all very new, as we get a feel for how this works we'll likely update, add groups, and define some ground rules. In the meantime, be respectful and let's try to make this a helpful discussion board.

**Comment by EmmanuelTejedaS** [comment]
### what happened to the angular fire documentation for ionic cli

I can't find the Installation and Setup with Ionic CLI

can't be used anymore?

**Comment by TEKbandrus** [comment]
Is the Q&A Board permanently closed? I'm unable to ask a new question (at least I can't find/figure out how/where I'm supposed to do this?)

**Comment by Jonesxxl** [comment]
Is this the way to fetch the user's uid after sign in? I want to get the uid to then fetch the user data with it.

`firebase.auth().signInWithEmailAndPassword(email, password)
  .then((userCredential) => {
    // Signed in 
    var user = userCredential.user;
    var uid = user.uid;
    // ...
  })
  .catch((error) => {
    var errorCode = error.code;
    var errorMessage = error.message;
    // ...
  });`


  
  
   I can only find examples on how to do it with the classic Firebase SDK but theres nothing on how to do it with AngularFire.
  
  Thanks in advance for your help :)

**Reply by pjanzen53** [reply]
I'm actually looking for the exact same answer to this. I have my login page and registration page working but I still for some reason can't get things to link to firestore for creating a user profile database

**Comment by galassimarco** [comment]
Can someone help me understand why there is an @angular/fire 7.6 and then @angular/fire 16?
Are they the same? What's the difference?

**Reply by jamesdaniels** [reply]
We ended up moving to have our major versions be the same as Angular's, so it's more clear which version we support. AngularFire v16 works with Angular v16.

**Comment by spock123** [comment]
Hey guys, we're trying to go zoneless for our Angular 17.1 application, but @angular/fire throws an error when zonejs is not present. Did any of you guys by chance try this and figured out if there is a workaround for this?

**Reply by jamesdaniels** [reply]
We're prioritizing support for zoneless very soon

**Reply by spock123** [reply]
Thank you so much

**Comment by niallmomo** [comment]
Howdy! Just checking to see if you fine chaps are working on a V21 version of AngularFire?

**Comment by PiotrKalachyn** [comment]
Hi, AngularFire people! How safe is it to use AngularFire v21-rc with Angular 22? Thank you for years of developing A/F!

```
    "@angular/core": "^22.0.2",
    "@angular/fire": "^21.0.0-rc.0",
```
